cpp/include/nvtext/edit_distance.hpp

```diff
@@ -50,8 +50,8 @@ namespace CUDF_EXPORT nvtext {
  * The `targets.size()` must equal `input.size()` unless `targets.size()==1`.
  * In this case, all `input` will be computed against the single `targets[0]` string.
  *
- * @throw cudf::logic_error if `targets.size() != input.size()` and
- *                          if `targets.size() != 1`
+ * @throw std::invalid_argument if `targets.size() != input.size()` and if `targets.size() != 1`
+ * @throw std::invalid_argument if `targets.size()==1` and `targets[0].is_null()`
```

Comment:
```suggestion
 * @throw std::invalid_argument if `targets.size()== 1` and `targets[0].is_null()`
```

Reply:
This is handled ok by the current logic and does not need to throw.

Reply:
Is your suggestion here to add a single space before the `1`?

Reply:
Yes, that suggestion (1) is independent from my question (2).

Reply:
Ok. I think I addressed both here.